src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -5200,6 +5200,12 @@ internal static List<CompletionResult> CompleteVariable(CompletionContext contex
 
             var lastAst = context.RelatedAsts?[^1];
             var variableAst = lastAst as VariableExpressionAst;
+            if (lastAst is PropertyMemberAst ||
+                (lastAst is not null && lastAst.Parent is ParameterAst parameter && (parameter.DefaultValue is null || parameter.DefaultValue != lastAst)))
```

Comment:
> (parameter.DefaultValue is null || parameter.DefaultValue != lastAst)

Can this be simplified to `parameter.DefaultValue != lastAst`? You have the check `lastAst is not null` in the beginning, so if `parameter.DefaultValue is null`, then `parameter.DefaultValue != lastAst` will be true.

Reply:
Yeah I think so. It's gone now.